cegs_portal/search/templates/search/v1/experiment.html

```diff
@@ -67,6 +76,15 @@
                     <div class="text-2xl font-bold flex items-center justify-center" style="color: rgb(71 85 105);">
                         <svg xmlns="http://www.w3.org/2000/svg" viewBox="0 0 640 512" class="svg-icon mr-2"><!--! Font Awesome Free 6.4.0 by @fontawesome - https://fontawesome.com License - https://fontawesome.com/license (Commercial License) Copyright 2023 Fonticons, Inc. --><path d="M175 389.4c-9.8 16-15 34.3-15 53.1c-10 3.5-20.8 5.5-32 5.5c-53 0-96-43-96-96V64C14.3 64 0 49.7 0 32S14.3 0 32 0H96h64 64c17.7 0 32 14.3 32 32s-14.3 32-32 32V309.9l-49 79.6zM96 64v96h64V64H96zM352 0H480h32c17.7 0 32 14.3 32 32s-14.3 32-32 32V214.9L629.7 406.2c6.7 10.9 10.3 23.5 10.3 36.4c0 38.3-31.1 69.4-69.4 69.4H261.4c-38.3 0-69.4-31.1-69.4-69.4c0-12.8 3.6-25.4 10.3-36.4L320 214.9V64c-17.7 0-32-14.3-32-32s14.3-32 32-32h32zm32 64V224c0 5.9-1.6 11.7-4.7 16.8L330.5 320h171l-48.8-79.2c-3.1-5-4.7-10.8-4.7-16.8V64H384z"/>
                         </svg> {{ experiment.name }}
+                        {% if analyses|length > 1 %}
+                            <select class="analysis-button border-2 p-2 text-sm" id="dropdown">
+                                {% for analysis in analyses %}
+                                    <option value="{% url 'search:experiment' experiment.accession_id %}?analysis={{analysis}}">V{{ forloop.counter }}</option>
```

Comment:
It's not necessarily the case that new analyses are updated versions of older analyses. We don't totally know how this could be used, but maybe there are two analyses looking at different aspects of the same experiment.

Reply:
Right, this was discussed and agreed upon Friday knowing that there could be many ways they can be used. I believe this was for simplicity. We can definitely discuss this further on Friday.